packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/create-database/create-database.controller.js

```diff
@@ -0,0 +1,51 @@
+import get from 'lodash/get';
+import capitalize from 'lodash/capitalize';
+
+export default class {
+  /* @ngInject */
+  constructor($translate, DatabaseService) {
+    this.$translate = $translate;
+    this.capitalize = capitalize;
+    this.DatabaseService = DatabaseService;
+  }
+
+  $onInit() {
+    this.trackDashboard('database::add', 'page');
+  }
+
+  cancel() {
+    this.goBack();
+  }
+
+  addDatabase() {
+    this.processing = true;
+    this.trackDashboard('database::add_validate');
+    return this.DatabaseService.addServiceDatabase(
+      this.projectId,
+      this.database.engine,
+      this.database.id,
+      this.databaseName,
+    )
+      .then((database) =>
+        this.goBack({
+          textHtml: this.$translate.instant(
+            'pci_databases_databases_create_database_success_message',
+            {
+              database,
+            },
+          ),
+        }),
+      )
+      .catch((err) =>
+        this.goBack(
+          this.$translate.instant(
+            'pci_databases_databases_create_database_error_message',
+            {
+              message: get(err, 'data.message', null),
```

Comment:
`err.data?.message || error.message`

Reply:
We'd rather not do it if possible as we don't feel it's consistent with how it's made elsewhere in the database pages.